tests/jit/test_cake_fused_moe_warp_decode_jit.py

```
@@ -0,0 +1,141 @@
+    monkeypatch.setattr(
+        cake_fused_moe_warp_decode,
+        "_get_compute_capability",
+        get_compute_capability,
+    )
+    monkeypatch.setattr(
+        cake_fused_moe_warp_decode,
+        "_build_and_load_cake_fused_moe_warp_decode_module",
+        fail_if_built,
+    )
+
+    with pytest.raises(RuntimeError, match="exact compute capability 10.3"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)